monkey/infection_monkey/example.conf

```diff
@@ -100,9 +103,9 @@
     "use_file_logging": true,
     "victims_max_exploit": 7,
     "victims_max_find": 30,
-    "post_breach_actions" : []
-    custom_PBA_linux_cmd = ""
```

Comment:
Nice find!

Reply:
Thanks :) , some of the key:value pair needs to be changed, happened to bump into them when I was using the config file to choose the pe which I wanted to run, happy coincidence haha :D